docs/docs/faq.md

```diff
@@ -0,0 +1,60 @@
+# Frequently asked questions
+
+### Why is the `NEBARI_KUBECONFIG` file in `/tmp`?
```

Comment:
Headers should follow a hierarchical ordering - H1 -> H2 -> H3 

here we are jumping from H1 -> H3, please fix